preload-webview.js

```diff
@@ -7,43 +7,47 @@ const PluginHost = remote.getGlobal('PluginHost')
 const currentWindow = remote.getCurrentWindow()
 const { app } = currentWindow.args
 
-const clientInterface = client => {
+const pluginInterface = plugin => {
   return {
-    name: client.name,
-    displayName: client.displayName,
-    type: client.type,
+    name: plugin.name,
+    displayName: plugin.displayName,
+    type: plugin.type,
+    api: plugin.api,
+    stdinWrite: payload => {
```

Comment:
Assuming this is motivated by Clef? Should it be included in this PR?

Reply:
thanks for catching